modules/ROOT/pages/configuring-dev-env.adoc

```diff
@@ -53,3 +54,191 @@ include::{partialsdir}/attributes.adoc[]
+        jcenter() // <-- Add This line
+        google()
+    }
+}
+----
+
+*Reference the Dependencies*
+
+In the build.gradle file of your app you can begin adding dependencies provided by the SDK.
+
+[source,groovy]
+----
+dependencies {
+    ...
+    implementation 'org.aerogear:android-push:[version]'
+    implementation 'org.aerogear:android-auth:[version]'
+    implementation 'org.aerogear:android-security:[version]'
+}
+----
+
+*Additional Dependencies*
+
+For push you also need to add a dependency to the Firebase SDK:
+
+[source,groovy]
+----
+dependencies {
+    ...
+    implementation "com.google.firebase:firebase-messaging:[version]"
+}
+----
+
+A full list of releases can be viewed https://github.com/aerogear/aerogear-android-sdk/releases[here]
+
+*Certificate Pinning*
+
+To implement certificate pinning in the individual SDKs, see link:certificate-pinning.html[the certificate pinning guide].
+
+****
+
+[role="secondary"]
+.iOS
+
+****
+
+*Prerequisites*
+
+To use the AeroGear SDK for iOS, you will need the following installed on your development machine:
+
+* iOS 9 or later
+* Xcode 9 or later
+* link:https://cocoapods.org/[CocoaPods]
+
+To get started with the AeroGear iOS SDK, you can set up the SDK in your existing or new project. The SDK is written in Swift.
+
+*CocoaPods*
+
+. The AeroGear SDK for iOS is available using the CocoaPods package manager.
+If you have not installed CocoaPods, install it by running the command:
++
+[source,bash]
+----
+gem install cocoapods
+pod setup
+----
++
+. In your project directory (the directory where your `.xcodeproj` file is), create a plain text file named `Podfile` (without any file extension) and add the lines below. Replace `YourTarget` with your actual target name.
++
+[source,ruby]
+----
+source 'https://github.com/CocoaPods/Specs.git'
++
+platform :ios, '9.0'
+use_frameworks!
+
+target :'YourTarget' do
+    ## Require one of the AeroGear SDK's here
+    pod 'AGSPush'
+    pod 'AGSAuth'
+end
+----
++
+. Then run the following command:
++
+[source,bash]
+----
+pod install
+----
++
+. Open up `.xcworkspace` with Xcode and start using the SDK.
++
+> Do not use `.xcodeproj`. If you open up a project file instead of a workspace, dependencies will not be correctly configured.
++
+. Import SDK you want to use in the application
+For example:
+
+[source,swift]
+----
+   import AGSAuth
+----
+
+****
+
+[role="secondary"]
+.Cordova
+
+****
+
+*Prerequisites*
+
+Before installing the SDK you will need in your system, at least:
+
+* link:https://nodejs.org/[Node.js] runtime and NPM
+* link:https://cordova.apache.org/docs/en/latest/guide/cli/#installing-the-cordova-cli[Cordova CLI]
+
+*Add AeroGear SDK*
+
+1. Get your app's configuration from the link:https://github.com/aerogear/mobile-cli[AeroGear Mobile CLI]. This JSON file contains custom confguration data for your project and shouldn't be changed manually. Every time a new service is provisioned the file has to be replaced by a new one.
```

Comment:
Not sure if we want tell mobile guys to use cli. Getting this from OpenShift UI will be much easier as they do not need to go with tons of setup required to run cli like installing oc tool etc.

Reply:
Also, I think the steps should be more concise. I already added this section as part of Metrics initialization in #51 and it looks like the image. I am more inclined of very straightforward instructions:
![screen shot 2018-06-15 at 11 53 23](https://user-images.githubusercontent.com/11672286/41462568-91fd7800-7093-11e8-9bc9-f07307ca621f.png)

Oh and also, this section is to describe how to install the SDK for each particular module, in this case Push. So  I think it's missing the `npm i @aerogear/push`.

Reply:
I'll use the Openshift method

Reply:
@josemigallas This is the generic document, so i don't describe how to install the push plugin. That is part of another document, coding.adoc inside the push folder.

Reply:
Fair enough, I would remove the config file part, it's duplicated.

Reply:
Ok